packages/thumbprint-react/package.json

```diff
@@ -9,7 +9,11 @@
     ],
     "sideEffects": false,
     "scripts": {
-        "prepublishOnly": "../../scripts/should-build-package.js -- rollup -c=./rollup.config.js"
+        "generateTypeDeclarations": "tsc --project ./tsconfig.types.json",
+        "buildJsFiles": "../../scripts/should-build-package.js -- rollup -c=./rollup.config.js",
+        "clear": "rm -rf dist",
+        "build": "yarn clear && yarn buildJsFiles && yarn generateTypeDeclarations && yarn generateTypeDeclarations --outDir ./dist/cjs",
+        "prepublishOnly": "yarn build"
```

Comment:
Would `../../scripts/should-build-package.js -- yarn build` work? That would help speed up `yarn start` (and other scripts) when the React package doesn't change.

Reply:
good idea, i can move that here